automatic/tightvnc/tightvnc.nuspec

```diff
@@ -1,8 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
-    <id>{{PackageName}}</id>
-    <version>{{PackageVersion}}</version>
+    <id>tightvnc</id>
+    <version>2.8.5</version>
```

Comment:
Don't set the latest version if it isn't available on chocolatey,
either set it to `0.0` or the latest version available on chocolatey.